packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/data_sources/remote/http_clients/api_http_client.dart

```diff
@@ -29,4 +40,18 @@ class ApiHttpClient with DioMixin implements Dio {
       analyticsInterceptor,{{/analytics}}
     ]);
   }
+  {{#enable_dev_menu}}
+
+  Future<void> setupProxy() async {
```

Comment:
I believe this function belongs to the dev menu lib/brick

Reply:
This function could be marked as private if it is not needed outside of the constructor. We do not want to expose new APIs from the http clients unless absolutely necessary.